Modules/Microsoft365DSC/DSCResources/MSFT_AADEntitlementManagementConnectedOrganization/MSFT_AADEntitlementManagementConnectedOrganization.psm1

```diff
@@ -313,8 +313,9 @@ function Set-TargetResource
                 $CreateParameters.$key = Convert-M365DSCDRGComplexTypeToHashtable -ComplexObject $CreateParameters.$key
             }
         }
-
-        $newConnectedOrganization = New-MgBetaEntitlementManagementConnectedOrganization -BodyParameter $CreateParameters
+        $TenantId = $CreateParameters.IdentitySources.ExternalTenantId
+        $DomainName = (Invoke-MgGraphRequest -Method 'GET' -Uri "https://graph.microsoft.com/beta/tenantRelationships/microsoft.graph.findTenantInformationByTenantId(tenantId='$tenantid')").defaultDomainName
```

Comment:
We should be hardcoding urls. Instead, for the FQDN, use the $Global:MSCloudLoginConnectionProfile.MicrosoftGraph.ResourceUrl variable. This is for sovereign clouds support where the url isn't graph.microsoft.com

Reply:
Also, can we try to use the cmdlet where possible instead of Invoke-MgGraphRequest. Thanks

Reply:
I have not found the cmdlet for this, sorry :-)